package.json

```diff
@@ -5,6 +5,12 @@
     "main": "index.js",
     "keywords": [],
     "author": "",
+    "repository": {
```

Comment:
We can actually just do “repository”:”eslint/new.eslint.org” here.

Reply:
done